config/exercise_readme.go.tmpl

```diff
@@ -18,6 +18,19 @@ To raise a message with an exception, just write it as an argument to the except
 raise Exception("Meaningful message indicating the source of the error")
 ```
 
+## Running the tests
+
+To run the tests run the command `python -m pytest {{ .Spec.SnakeCaseName }}_test.py`
+(`py.test {{ .Spec.SnakeCaseName }}_test.py` for Python 2; `pytest <exercise>_test.py` for Python 3). Or run it with
```

Comment:
This should be `{{ .Spec.SnakeCaseName }}` instead of `<exercise>_test.py` for Python 3 as well.